stable/mysql/README.md

```diff
@@ -71,6 +71,10 @@ The following table lists the configurable parameters of the MySQL chart and the
 | `persistence.subPath`                | Subdirectory of the volume to mount       | `nil`                                                |
 | `resources`                          | CPU/Memory resource requests/limits       | Memory: `256Mi`, CPU: `100m`                         |
 | `configurationFiles`                 | List of mysql configuration files         | `nil`                                                |
+| `ssl.enabled`                        | Setup and use SSL for MySQL connections   | `false`                                              |
+| `ssl.ca`                             | CA certificate                            | `nil`                                                |
+| `ssl.cert`                           | Server certificate (public key)           | `nil`                                                |
+| `ssl.key`                            | Server key (private key)                  | `nil`                                                |
```

Comment:
The wordpress chart is an example of one that does ssl and secrets with a slightly different pattern (]you can see it here](https://github.com/kubernetes/charts/blob/master/stable/wordpress/values.yaml#L170)). The idea is that setting the secret to use is different from passing in the values and creating a secret. This way you can use an outside secret, not managed, by the chart if you want.